client/my-sites/plugins/plugins-list/index.jsx

```diff
@@ -131,12 +131,12 @@ export default React.createClass( {
 		const bulkManagement = ! this.state.bulkManagement;
 
 		if ( bulkManagement ) {
+			this.setBulkSelectionState( this.props.plugins, bulkManagement );
 			this.setState( { bulkManagement } );
 			return this.recordEvent( 'Clicked Manage' );
 		}
 
-		// Unselect all plugins.
-		this.setState( { selectedPlugins: {}, bulkManagement } );
```

Comment:
`selectedPlugins` seem like a fair name to me. I can't think of any other names that would lend any more clarity. 

I do think of a collection when reading the name so perhaps an ES6 Map would be more appropriate than using an Object. Although historically an Object is appropriate